homeassistant/components/abode/__init__.py

```diff
@@ -179,27 +186,23 @@ def capture_image(call):
         """Capture a new image."""
         entity_ids = call.data.get(ATTR_ENTITY_ID)
 
-        target_devices = [
-            device
-            for device in hass.data[DOMAIN].devices
-            if device.entity_id in entity_ids
+        target_entities = [
+            entity for entity in hass.data[DOMAIN].entity_ids if entity in entity_ids
         ]
 
-        for device in target_devices:
-            device.capture()
+        for entity in target_entities:
+            dispatcher_send(hass, SIGNAL_CAPTURE_IMAGE)
```

Comment:
Create a specific signal by using string formatting and passing the entity_id as the format variable.
```py
signal = SIGNAL_CAPTURE_IMAGE.format(entity_id)
dispatcher_send(hass, signal)
```